### DISCUSSION TITLE
Ability to Import SQL files or export entire DB in SQL

### DISCUSSION THREAD

**Discussion by silicahd** [outerbase/studio#178] [Ideas]
I guess the title is mostly the idea. I am trying to import a large db in a D1 sql database but there is no way to import it. I guess i can just copy paste the entire code but would be nice to have a file import. 

Also I would love to pay for a backup service. In short offer a backup service for a price where each connection is backed up at a set time. Allow users to compare a backup to another backup or live db. These would be worth paying for.

**Comment by invisal** [comment]
How large is your database?

**Reply by silicahd** [reply]
748kb so not large but i am trying to load a mysql to sqllite so that might be my issue. However i am trying to only load the data as the schema was already created by my application.

**Reply by invisal** [reply]
Oh it is not that big. Let me add this in our roadmap. I thought it is several GB and it would be hard to make it work inside the browser given that everything need to load in memory.
